.eslintrc.json

```diff
@@ -1,47 +0,0 @@
-{
-    "parserOptions":
-    {
-        "ecmaVersion": 2017
-    },
-    "plugins": [
-        "jest"
-    ],
-    "rules": {
```

Comment:
I would really prefer we re-add the rules around formatting of curly braces, white spacing, and variables. Do you have any objection to keeping them?

Reply:
I don't. I'll add them back.

Reply:
I added them back.